client/state/happychat/selectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { map, head, get } from 'lodash';
+import { get, head, map } from 'lodash';
```

Comment:
Are these automated changes? If so, which tool? I have `eslint` set up to fix most things for me but it doesn't alphabetize imports.